internal/bminventory/inventory.go

```diff
@@ -73,6 +73,7 @@ var (
 	DefaultClusterNetworkCidr       = "10.128.0.0/14"
 	DefaultClusterNetworkHostPrefix = int64(23)
 	DefaultServiceNetworkCidr       = "172.30.0.0/16"
+	DefaultNTPSource                = "clock.redhat.com"
```

Comment:
`clock.redhat.com` only exists inside the Red Hat VPN, so we need to have a different default value. RHEL defaults to:

```
0.rhel.pool.ntp.org
1.rhel.pool.ntp.org
2.rhel.pool.ntp.org
3.rhel.pool.ntp.org
```

If you're looking for a single value as opposed to a list, I suggest `0.rhel.pool.ntp.org` to follow precedent.